### PR TITLE
created category model

### PR DESCRIPTION
**Pull request by ThePrevailingOne**
Partial solve to #21

### CODE REVIEW

File: src/db/migrations/20210307133247-create-category.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable('Categories');
+  }
+};
```
Comment (ryoarmanda) [inline src/db/migrations/20210307133247-create-category.js:27]: Can add a blank line after the last line here for convention

**Review by ryoarmanda — CHANGES_REQUESTED** [review]
Good job! All's good from me, but can you bring back the `.env.sample` file? It shouldn't be removed as it is a sample file for developers.

**Comment by ryoarmanda** [comment]
Heads up that you need to pull the main branch, merge it to your PR branch, and resolve the conflicts first

**Comment by ThePrevailingOne** [comment]
Oh right noted!

**Comment by ThePrevailingOne** [comment]
Let me know kalo ada yg salah wkwkwk

File: src/db/migrations/20210307133247-create-category.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Categories', {
```
Comment (ryoarmanda) [inline src/db/migrations/20210307133247-create-category.js]: Can you name the table in lowercase style? So it's `categories`

File: src/db/migrations/20210307133247-create-category.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        type: Sequelize.DATE
+      },
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable('Categories');
```
Comment (ryoarmanda) [inline src/db/migrations/20210307133247-create-category.js]: Same here

**Review by ryoarmanda — CHANGES_REQUESTED** [review]

**Comment by ThePrevailingOne** [comment]
done!

**Review by ryoarmanda — APPROVED** [review]
LGTM!

**Pull request merged**